packages/plugin-hardhat/test/transparent-admin-initial-owner.js

```diff
@@ -0,0 +1,62 @@
+const test = require('ava');
+
+const { ethers, upgrades } = require('hardhat');
+const hre = require('hardhat');
+
+test.before(async t => {
+  t.context.Greeter = await ethers.getContractFactory('Greeter');
+});
+
```

Comment:
What about adding this variable and replacing every occurrence?

```suggestion
const OWNABLE_ABI = ['function owner() view returns (address)']
```